torch/__init__.py

```diff
@@ -268,6 +268,8 @@ def typename(o):
 
     return module + class_name
 
+def is_bf16_support():
```

Comment:
this should not be in the `torch` namespace (for cpu, the criteria for bfloat16 support are different), please put it in torch.cuda, also call it `is_bf16_supported()`

Reply:
ok

Reply:
can we please make it more readable? Either:

1. `is_bf16_supported`
2. `has_bf16_support`

I vote for (1)

Thank you!